bcgov/HOUS-permit-portal
# Feature/bphh 1462/view accepted eula

## Description
- Add an EULA section and past EULA modal in user profile screen.
- Updates EULA copy based on Jira card


## What type of PR is this? (check all applicable)


## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-1462


## Steps to QA
- Pre req: ensure db and data migrations are run
- 

https://github.com/user-attachments/assets/74644ce2-7ee6-41d0-8c53-d6f3330d7f3b
